LayoutTests/fast/images/reset-image-animation.html

```diff
@@ -16,12 +16,12 @@
                 shouldBe("internals.imageFrameIndex(image)", expectedFrame.toString());
                 setTimeout(() => {
                     resolve(expectedFrame + 1);
-                }, 30);
+                }, 300);
```

Comment:
I do not think this is the right approach. Please try this one:

1. Add `image` to the document so its renderers can receive events:
```
    var image = new Image;
    document.body.appendChild(image);
```
2. Make the image send notifications to the renderers when it finishes decoding a frame:
```
    (function() {
        if (window.internals) {
            internals.clearMemoryCache();
            internals.settings.setWebkitImageReadyEventEnabled(true);
        }
        ...
    })();
```
3. Replace this setTimeout with this eventListener
```
    // Wait for the next image frame to finish decoding before moving foreward.
    image.addEventListener("webkitImageFrameReady", function() {
        resolve(expectedFrame + 1);
    }, false);
```

Reply:
Good idea. That's what I want. Will fix. Thank you.

Reply:
Fixed.